ros-conversion/cpp_message/CMakeLists.txt

```diff
@@ -58,37 +63,43 @@ ament_auto_add_library(${node_lib} SHARED
         )
 
 
-add_library(testlib SHARED IMPORTED)
+ament_target_dependencies(${node_lib}
```

Comment:
This can be removed, class_loader is not used and rclcpp and rclcpp_components dependencies should be found by ament_auto_find_build_dependencies